### PR TITLE
eslint: fix broken rc file

### PR DESCRIPTION
**Pull request by MarcoWorms**
@mccraveiro using airbnb without base was exploding this config. Do you think we can use this styleguide for non-react projects and then use @derekstavis styleguide for react that he is writing atm?

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,17 +1,17 @@
-{
-  "extends": "airbnb",
-  "rules": {
-    "semi": "never",
-    "space-before-function-paren": [2, "always"],
-    "comma-dangle": [
+module.exports = {
+  extends: ['airbnb-base'],
+  rules: {
+    'semi': [2, 'never'],
```
Comment (mccraveiro) [inline .eslintrc]: remove quotes around `semi` as they are not needed

**Comment by mccraveiro** [comment]
@MarcoWorms Yeap! I do agree that it would be better if we split this style guide in two.
Could you create the repo for `react-style-guide`?

**Comment by MarcoWorms** [comment]
IMO we should centralize styleguides/standards in one repo

**Comment by MarcoWorms** [comment]
Also, I use the JS format because JSON spec is poop (no dangling comma for instance)

**Pull request merged**